packages/core/src/awsService/appBuilder/serverlessLand/serverlessLandMain.ts

```diff
@@ -0,0 +1,88 @@
+/*!
```

Comment:
Considering that we everything is inside `serverlessLand/` at this point I would just call the files `main.ts`, `wizard.ts`, etc.